docs/theming/developers.md

```diff
@@ -35,24 +35,24 @@ hasToc: true
 
 ## Using the theming system
 
-<abbr title="Red Hat Design System">RHDS</abbr>' theming system is a high-level 
+<abbr title="Red Hat Design System">RHDS</abbr>’ theming system is a high-level 
 expression of the lower-level components of the system, [tokens][tokens], and 
-[elements][elements], and in turn, it factors into the development of the 
-highest level design system component, [patterns][patterns]. In order to use 
+[elements][elements]. In turn, it factors into the development of the 
+highest-level design system component, [patterns][patterns]. To use 
```

Comment:
If you do add the colon above, add one here, too:

`the highest-level design system component: [patterns][patterns].`

Reply:
[8aa6731](https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/8aa6731ac3ce8c1716894fc135062a58ed031088)